build.sbt

```diff
@@ -15,7 +15,7 @@ val googleClientApiV = "1.20.0"
 
 libraryDependencies ++= Seq(
   "com.gettyimages" %% "spray-swagger" % "0.5.1",
-  "org.webjars" % "swagger-ui" % "2.0.24",
+  "org.webjars" % "swagger-ui" % "2.1.8-M1",
```

Comment:
Apparently a "stable" release has been released _after_ 2.1.8-M1 that I missed, due to the way versions are sorted. "Updated" to 2.1.1.